OpenGpxTracker/ViewController.swift

```diff
@@ -188,39 +222,78 @@ class ViewController: UIViewController, UIGestureRecognizerDelegate  {
     
     
     //UI
-    //labels
+    /// Label with the title of the app
     var appTitleLabel: UILabel
-    //var appTitleBackgroundView: UIView
+
+    /// Image with the GPS signal
     var signalImageView: UIImageView
+    
+    /// Current GPS signal accuracy text (based on kSignalAccuracyX constants)
     var signalAccuracyLabel: UILabel
+    
+    /// Label that displays current latitude and longitude (lat,long)
     var coordsLabel: UILabel
+    
+    /// Displays current elapsed time (00:00
     var timeLabel: UILabel
+    
+    /// Label that displays last known speed (in km/h)
     var speedLabel: UILabel
+    
+    /// Distance of the total segments tracked
+    ///
+    /// Displayed in m or km based on the
```

Comment:
incomplete `/// Displayed in m or km based on the`

Same as for var `currentSegmentDistanceLabel`

Reply:
fixed in 2511fdfc4f8b59f2a1d462f22d5355a64b67ae9e